ionic2/app/pages/wiki-index/wiki-index.page.scss

```diff
@@ -0,0 +1,290 @@
+.wiki-index {
+  background-color: #f4f4f7;
+  .pokeimg-fixed {
+    width: 100px;
+    height: 100px;
+  }
+  .card-center {
+    display: flex;
+    justify-content: center;
+  }
+}
+
+
+
+/* ==========================================================================
+    GRIDLEX
+    Just a Flexbox Grid System
```

Comment:
It's not the only way, but I can't be fucked to write a column type system on my own so I figured it would be best to import it. Unless someone wants to do it himself everyone can contribute ;)